### ISSUE TITLE
Consider Heroku Review apps for deploying branches

### ISSUE THREAD

**Issue by shaunakpp** [education/classroom#2224]
With the recent update causing us to go into maintenance mode for a while, we can consider enabling [review apps](https://devcenter.heroku.com/articles/github-integration-review-apps) 
for Classroom. It can help us deploy branches in a production-like environment, without affecting the availability of Classroom.

cc: @d12 @andrewbredow

**Comment by andrewbredow** [comment]
@shaunakpp Agreed... as long as we're in Heroku-land, we should take advantage of review apps. Adding to our backlog for operationalizing.
